### PR TITLE
Optimize doc-releases: Schedule once per day instead of per release

### PR DESCRIPTION
**Pull request by a-hilaly**
To avoiid exhausting GitHub API quota when releasing multiple controllers
in parallel, the docsrelease script will now run once every 24 hours instead
of on "every controller release". With 42 controllers, the previous approach
was leading to excessive GitHub API requests due to the script crawling
and high API usage. This change optimizes the process while maintaining
regular doc updates.

Signed-off-by: Amine Hilaly <hilalyamine@gmail.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prow/jobs/jinja/periodics/docs_release.jinja2 [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: docs-release
+  decorate: true
+  interval: 6h
```
Comment (acornett21) [inline prow/jobs/jinja/periodics/docs_release.jinja2:3]: Shouldn't this be 24h?

**Review by acornett21 — COMMENTED** [review]
Along with the question on one of the files, will this help to increase the threshold of other jobs/processes that were also experiencing rate-limiting?

File: prow/jobs/jinja/periodics/docs_release.jinja2 (hunk 1 above)
Comment (a-hilaly) [reply prow/jobs/jinja/periodics/docs_release.jinja2]: Good catch! thank you sir! :)

**Comment by a-hilaly** [comment]
>will this help to increase the threshold of other jobs/processes that were also experiencing rate-limiting?

@acornett21 correct, this will leave some air for other prow jobs (e.g. OLM scripts) to execute without hitting API rate limits

**Comment by a-hilaly** [comment]
Note to reviewers: We will still be able to execute this job on-demand, if we don't want to wait 24hours...

File: prow/jobs/jinja/periodics/docs_release.jinja2 (hunk 1 above)
Comment (acornett21) [reply prow/jobs/jinja/periodics/docs_release.jinja2]: I believe there is 24h, in another place as well.

Comment (a-hilaly) [reply prow/jobs/jinja/periodics/docs_release.jinja2]: Yes we have other jobs that are scheduled to run every 6hours (label_sync, github issues stale/rotten plugins)

Comment (acornett21) [reply prow/jobs/jinja/periodics/docs_release.jinja2]: Thanks for explaining, just wanted to confirm this is now 💯

**Pull request merged**